api/src/opentrons/protocol_engine/state/geometry.py

```diff
@@ -430,6 +433,20 @@ def get_well_position(
                 offset = offset.copy(update={"z": offset.z + well_depth})
             elif well_location.origin == WellOrigin.CENTER:
                 offset = offset.copy(update={"z": offset.z + well_depth / 2.0})
+            elif well_location.origin == WellOrigin.MENISCUS:
+                height = self._wells.get_last_measured_liquid_height(
+                    labware_id, well_name
+                )  # in deck coordinates
+                if height:
```

Comment:
```suggestion
                if height is not None:
```

Nitpick: Let's avoid a height of `0.0` counting as "liquid height unknown," since `get_last_measured_liquid_height()` defines `None` as the official sentinel for "liquid height unknown".

This probably will not matter in practice because the probe will probably never return a height of exactly 0.0. Just a semantic thing.

Reply:
Good call. Changed this population to None so this works: https://github.com/Opentrons/opentrons/blob/5e8fbedcb0213677c377c50c70ab7075def62f83/api/src/opentrons/protocol_engine/state/wells.py#L55